index.html

```diff
@@ -6,14 +6,32 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Alicante Frontend</title>
+    <meta name="description" content="El ecosistema Frontend es un caos y ¡nos encanta! Si quieres aprender y compartir sobre desarrollo web orientado al Frontend has llegado al sitio correcto.">
     <link rel="icon" type="image/png" sizes="32x32" href="src/img/favicon-32x32.png">
     <link rel="icon" type="image/png" sizes="16x16" href="src/img/favicon-16x16.png">
     <link rel="shortcut icon" href="src/img/favicon.ico">
     <link rel="apple-touch-icon" sizes="180x180" href="src/img/apple-touch-icon.png">
     <link rel="mask-icon" href="src/img/safari-pinned-tab.svg" color="#3a84d0">
     <link rel="manifest" href="src/img/manifest.json">
+    <link rel="stylesheet" href="src/css/style.css">
     <meta name="msapplication-config" content="src/img/browserconfig.xml">
     <meta name="theme-color" content="#3a84d0">
+    <meta name="twitter:card" content="summary">
+    <meta name="twitter:title" content="Alicante Frontend">
+    <meta name="twitter:description" content="El ecosistema Frontend es un caos y ¡nos encanta! Si quieres aprender y compartir sobre desarrollo web orientado al Frontend has llegado al sitio correcto.">
+    <meta name="twitter:site" content="@AlicanteFront">
+    <meta name="twitter:creator" content="@AlicanteFront">
+    <meta name="twitter:image:src" content="src/img/alicante-frontend-logo.svg">
```

Comment:
It looks like twitter's [image meta is just `twitter:image`](https://developer.twitter.com/en/docs/tweets/optimize-with-cards/overview/summary). The card validator doesn't seem to pick it up
![image](https://user-images.githubusercontent.com/5701162/31223804-caadd30a-a9cb-11e7-9b99-3bf574d855e8.png)

Additionally, the image displayed on the summary twitter card will be squared. Let's better use for twitter the [squared image (download from this link)](https://drive.google.com/file/d/0B7TJZZeLbeD_VGpGSWd4d3Zmc1k/view?usp=sharing)

With these two changes, twitter's should be ok

Reply:
According to [Twitter docs](https://developer.twitter.com/en/docs/tweets/optimize-with-cards/guides/getting-started#twitter-cards-and-open-graph), Twitter reads Open Graph tags, so no need to repeat some specific Twitter meta tags. With the following meta tags we'll have covered Google, Facebook, Twitter, Pinterest, etc. with different images for each one. Please, review the image urls in case I'm wrong.

PS: the world of metatags is awesome.

```
<!-- Search Engines -->
<title>Alicante Frontend</title>
<meta name="description" content="El ecosistema Frontend es un caos y ¡nos encanta! Si quieres aprender y compartir sobre desarrollo web orientado al Frontend has llegado al sitio correcto.">
<meta name="image" content="src/img/alicante-frontend.png">

<!-- Schema.org for Google -->
<meta itemprop="name" content="Alicante Frontend">
<meta itemprop="description" content="El ecosistema Frontend es un caos y ¡nos encanta! Si quieres aprender y compartir sobre desarrollo web orientado al Frontend has llegado al sitio correcto.">
<meta itemprop="image" content="src/img/alicante-frontend.png">

<!-- Twitter specific -->
<meta name="twitter:card" content="summary" />
<meta name="twitter:site" content="@AlicanteFront" />
<meta name="twitter:creator" content="@AlicanteFront" />
<meta name="twitter:image" content="src/img/alicante-frontend-tw.png"/>

<!-- Twitter, Facebook, Pinterest, etc. -->
<meta property="og:site_name" content="Alicante Frontend">
<meta property="og:url" content="http://www.alicantefrontend.es" />
<meta property="og:title" content="Alicante Frontend" />
<meta property="og:description" content="El ecosistema Frontend es un caos y ¡nos encanta! Si quieres aprender y compartir sobre desarrollo web orientado al Frontend has llegado al sitio correcto." />
<meta property="og:image" content="src/img/alicante-frontend-fb.png">
<meta property="og:locale" content="es_ES">
```